Cargo.toml

```diff
@@ -2,6 +2,9 @@
 members = [
     'node',
     'pallets/template',
```

Comment:
if this template pallet not used anymore, we can remove it. (In separate PR or same)